### PR TITLE
Reformated experiments headings and subheadings

### PR DESCRIPTION
**Pull request by NamanRawat1**
Fixes issue #634 
Apk-
[app-debug.zip](https://github.com/fossasia/pslab-android/files/1345479/app-debug.zip)

Screenshots for the change: 
screenshot_2017-09-30-00-54-48-337_org fossasia pslab

### CODE REVIEW

**Comment by NamanRawat1** [comment]
@viveksb007   Is this fine?

File: app/src/main/assets/DOC_HTML/apps/D_NFET.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+### Dependence of drain current on D-S voltage, and Gate Voltage
 
 * Set the gate voltage for various values between 0, and -2 V , and observe the effect on saturation current
 
 * PV1 is varied in steps, and for each step the current is calculated from the difference between voltages at PV1 and CH1, and the known value of the resistor(560 ohms)
 
 * Acquired plots can be selectively displayed or deleted.
 
-## screenshot
+### screenshot
```
Comment (viveksb007) [inline app/src/main/assets/DOC_HTML/apps/D_NFET.md]: Please keep it constant across all experiment files. If its `###` or `####`, it should be same across all files like a pre-defined format.

Comment (viveksb007) [reply app/src/main/assets/DOC_HTML/apps/D_NFET.md]: @NamanRawat1 Please check other headings too and ensure heading, sub-heading, etc. are same across all experiment files.

Comment (NamanRawat1) [reply app/src/main/assets/DOC_HTML/apps/D_NFET.md]: ok, I will make same across all files

**Comment by NamanRawat1** [comment]
Apk updated-[app-debug.zip](https://github.com/fossasia/pslab-android/files/1359061/app-debug.zip)
Headings-##
SubHeadings-###
OtherHeadings-####
Exception at some places based on experiments.

**Comment by mariobehling** [comment]
@viveksb007 Please review again.

**Pull request merged**